charts/pega/README.md

```diff
@@ -243,6 +243,32 @@ Parameter       | Description    | Default value
 `initialHeap`   | This specifies the initial heap size of the JVM.  | `4096m`
 `maxHeap`       | This specifies the maximum heap size of the JVM.  | `7168m`
 
+### Liveness and readiness probes
+
+[Probes are used by Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) as a way to determine if an application is healthy.  They may be configured for liveness to determine if a Pod has entered a broken state, or for readiness to determine if the application is available to be exposed.  Probes may be configured for each tier independently.  If not explicitly configured, default probes will be used.  The following parameters may be used as part of a `livenessProbe` or `readinessProbe` configuration.
+
+Parameter           | Description    | Default value
+---                 | ---            | ---
+`initialDelaySeconds` | Number of seconds after the container has started before liveness or readiness probes are initiated. | `300`
+`timeoutSeconds`      | Number of seconds after which the probe times out. | `20`
+`periodSeconds`       | How often (in seconds) to perform the probe. Some providers such as GCP require this value to be greater than the timeout value. | `30`
```

Comment:
Thanks looks good! GCP is fine since GKE will let you apply the probe but the error is coming from compute engine trying to create the health checks for the load balancer.